src/make_dbg_constants.py

```diff
@@ -12,10 +12,10 @@
 #
 # The input format is:
 #
-#	cmd: [DebugCmd]
-#	names: [space delimited names of cmd]
-#	resume: ['true' or 'false': should execution resume after this command?]
-#	help: [some help text]
+# cmd: [DebugCmd]
+# names: [space delimited names of cmd]
+# resume: ['true' or 'false': should execution resume after this command?]
+# help: [some help text]
```

Comment:
Can we mark up this as a code block, either by prefixing with four spaces or by wrapping in triple backticks?